sympy/integrals/prde.py

```diff
@@ -773,15 +773,15 @@ def limited_integrate_reduce(fa, fd, G, DE):
     c = reduce(lambda i, j: i.lcm(j), (dn,) + En)  # lcm(dn, en1, ..., enm)
     hn = c.gcd(c.diff(DE.t))
     a = hn
-    b = -derivation(hn, DE)
+    b = derivation(hn, DE)
```

Comment:
I would have preserved the minus sign here. It comes from the pseudocode in the book.

Reply:
Sure! No problems. I only removed it because of the incorrect reuse.